### ISSUE TITLE
Thrown grenades no longer explode after alt tabbing and returning in editor testing

### ISSUE THREAD

**Issue by soledad630** [Regalis11/Barotrauma#10934]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://user-images.githubusercontent.com/86010664/212463398-650d1d52-8280-4646-8944-bf75a32ff11e.png)
Under said setup, with windows 10 and discrete graphics mode on a 3060 laptop GPU, throwable grenades would explode after being thrown in a fresh editor testing mission, but not after alt tabbing out and wait for around 10-20s and come back. Acid grenades would weirdly emit acid once but has no expended durability, and stop working after being picked up showing full durability, other grenades will just lay there after being thrown with no effects whatsoever.

### Reproduction steps

1. Start a editor testing session.
2. Alt tab, wait for 10-30s, alt tab back into the game.
3. Throw some throwable grenades.

### Bug prevalence

Happens regularly

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I'm afraid I could not find a way to reproduce this. Alt-tabbing doesn't pause the game, and the grenades seem to explode normally when I'm alt-tabbed.

**Comment by Regalis11** [comment]
Got the game to pause now, but I was still unable to reproduce the issue. The grenades seem to explode with no issues once I unpause the game.

**Comment by soledad630** [comment]
Oh I meant after alt tabbing and alt tabbing back into the game, the greandes seems to no longer explode. I tried for a shorter duration of alt tabbing and sometimes it  will happen.
I went to check on if it's a sub size issue, so I recorded this current insance in barsuk.

This is a recording of a post-alt-tab behaviour on grenades.

https://user-images.githubusercontent.com/86010664/212806000-07321093-f05a-4e57-91ce-62c5e23e0de0.mp4

Due to length restriction I probably cannot record both the pre-alt-tab and post-alt-tab.

![Untitled](https://user-images.githubusercontent.com/86010664/212804822-0d282a99-a0bd-40e9-a805-592f42ed9a3c.png)
I also forgot to include that I disabled full screen optimization previously.
